msg/AirspeedWind.msg

```diff
@@ -1,22 +1,28 @@
+# Wind estimate (from WindEstimator)
+#
+# Contains local wind estimation and airspeed innovation information produced by the WindEstimator.
+# This message is used only internally by the airspeed selector and validator module
+# to assess airspeed measurement validity.
+
 uint64 timestamp		# time since system start (microseconds)
```

Comment:
only one bit of consecutive whitespace allowed. Capitalize first letters. So this would be
```suggestion
uint64 timestamp # [us] Time since system start
```